dials/views.py

```diff
@@ -0,0 +1,29 @@
+from django.shortcuts import render
+from django.shortcuts import get_object_or_404, render_to_response
+from django.template import RequestContext
+from models import Dial
+
+# Create your views here.
+def dial_svg(request, slug):
+    dial = get_object_or_404(Dial, slug=slug)
+    if dial.precent < 15:
+        color = "#ED1C24"
+    elif dial.precent < 40:
+        color = "orange" #TODO: pick a color
+    else:
+        color = "green" #TODO: pick a color
+    return render_to_response("dials/dial.html",
+            {'width': dial.precent * 746 / 100,
```

Comment:
@daonb Please change "magic numbers" to named constants, also is this always the same width? in mobile screens for example?